### ISSUE TITLE
Consistent crash when loading certain savegames

### ISSUE THREAD

**Issue by EvanDurfee** [TheDivexz/excelmakesbelhappypack#126]
I've tried playthroughs as Otomo and Tokugawa recently, and have run into an issue which will consistently crash my game for a given save. Sometimes it's worked to go back to an old enough save and the crash doesn't occur.
For instance, right now the crash occurs at the end of may on my save from January first, and once I saw the event window pop up right before the crash.
mod version: df5a169b819e3f290386e74df1322bea4b6c2827 (April 5)
EU4 version: 1.30.6

Only other mods I have active are cosmetic, and the crash still occurs with them disabled.

Stack trace:

>  Application: EUIV 
> Version: EU4 v1.30.6.0 Austria.dcdce3e4e4 
> Date/Time: 2021-05-06 21:46:25
> 
> Caught signal 11 (SIGSEGV)
> 
> Original Stack Trace:
>   ./eu4(_ZNK11CAndTrigger8EvaluateER11CEventScope+0xa) [0x160d21a]
>   ./eu4(_ZNK16CEstatePrivilege7IsValidER11CEventScopeP7CString+0x64) [0x1269888]
>   ./eu4(_ZN7CEstate16UpdatePrivilegesE11CCountryTag+0xda) [0x125d060]
>   ./eu4(_ZN8CCountry13UpdateEstatesEb+0x6d7) [0xeaa865]
>   ./eu4(_ZN8CCountry17DailyUpdateSerialEv+0xfec) [0xedb356]
>   ./eu4(_ZN10CGameState11DailyUpdateEv+0x5d4) [0x128b0e8]
>   ./eu4(_ZN10CGameState15HandleDailyTickEv+0x856) [0x128a630]
>   ./eu4(_ZNK17CDailyTickCommand7ExecuteEv+0x8e) [0x15e0650]
>   ./eu4() [0x1dcca2d]
>   ./eu4(_ZN8CSession15ProcessCommandsEv+0x236) [0x1dcc578]
>   ./eu4(_ZN8CSession6UpdateEv+0xfa) [0x1dcbd94]
>   ./eu4(_ZN15CEU4Application6UpdateEv+0x63) [0xe063c5]
>   ./eu4(_ZN12CInGameIdler4IdleEb+0x3f6) [0x131f79a]
>   ./eu4(_ZN12CApplication14UpdateOneFrameEb+0x12d) [0x1d87015]
>   ./eu4(_ZN12CApplication3RunEv+0x27) [0x1d874d7]
>   ./eu4(main+0x1b50) [0x1361200]
>   /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf2) [0x7f6f5aca0cb2]
>   ./eu4(_start+0x2a) [0xcd82ba]
> 
> Demangled Stack Trace:
>   ./eu4                          ( CAndTrigger::Evaluate(CEventScope&) const  + 0xa   )  [0x160d21a]
>   ./eu4                          ( CEstatePrivilege::IsValid(CEventScope&, CString*) const  + 0x64  )  [0x1269888]
>   ./eu4                          ( CEstate::UpdatePrivileges(CCountryTag)    + 0xda  )  [0x125d060]
>   ./eu4                          ( CCountry::UpdateEstates(bool)             + 0x6d7 )  [0xeaa865]
>   ./eu4                          ( CCountry::DailyUpdateSerial()             + 0xfec )  [0xedb356]
>   ./eu4                          ( CGameState::DailyUpdate()                 + 0x5d4 )  [0x128b0e8]
>   ./eu4                          ( CGameState::HandleDailyTick()             + 0x856 )  [0x128a630]
>   ./eu4                          ( CDailyTickCommand::Execute() const        + 0x8e  )  [0x15e0650]
>   ./eu4                          (                                                   )  [0x1dcca2d]
>   ./eu4                          ( CSession::ProcessCommands()               + 0x236 )  [0x1dcc578]
>   ./eu4                          ( CSession::Update()                        + 0xfa  )  [0x1dcbd94]
>   ./eu4                          ( CEU4Application::Update()                 + 0x63  )  [0xe063c5]
>   ./eu4                          ( CInGameIdler::Idle(bool)                  + 0x3f6 )  [0x131f79a]
>   ./eu4                          ( CApplication::UpdateOneFrame(bool)        + 0x12d )  [0x1d87015]
>   ./eu4                          ( CApplication::Run()                       + 0x27  )  [0x1d874d7]
>   ./eu4                          ( main                                      + 0x1b50)  [0x1361200]
>   /lib/x86_64-linux-gnu/libc.so.6 ( __libc_start_main                         + 0xf2  )  [0x7f6f5aca0cb2]
>   ./eu4                          ( _start           

Not sure how feasible it is to debug something like this, but if you have any ideas where to look / if you need more info let me know.

**Comment by EvanDurfee** [comment]
So, after further testing and crashing the game many times, I found one workaround was to remove any one of the non-monopoly burgher privileges, or several of the privileges of the nobles / church (I couldn't find a particular pattern). Then, no event fired at the end of may and my game kept running. I attached a screencap with the active privileges.
![Screenshot from 2021-05-06 22-27-13](https://user-images.githubusercontent.com/20136084/117361922-ea1f4500-aeba-11eb-99a2-9ab20e16ac12.png)
